implementations/rust/ockam/ockam_api/src/cloud/share/create.rs

```diff
@@ -26,11 +24,11 @@ pub struct CreateServiceInvitation {
     #[n(3)] pub recipient_email: String,
 
     // TODO: Should route be a MultiAddr?
-    #[n(4)] pub project_identity: IdentityIdentifier,
+    #[n(4)] pub project_identity: String,
     #[n(5)] pub project_route: String,
-    #[n(6)] pub project_authority_identity: IdentityIdentifier,
+    #[n(6)] pub project_authority_identity: String,
     #[n(7)] pub project_authority_route: String,
-    #[n(8)] pub shared_node_identity: IdentityIdentifier,
+    #[n(8)] pub shared_node_identity: String,
     #[n(9)] pub shared_node_route: String,
```

Comment:
This change is not at all desirable, but I was having obtuse error messages during Tauri's frontend `invoke` call rejecting the IdentityIdentifier fields for values I knew were otherwise valid, because I'd gotten some or all of them directly from an existing one. Would love to take a little time to troubleshoot that further.